doc/api/fs.md

```diff
@@ -1583,6 +1583,29 @@ fs.open('<directory>', 'a+', (err, fd) => {
 });
 ```
 
+Some characters are reserved under Windows which documented by
+[Naming Files, Paths, and Namespaces][].
+
++ `<` (less than)
```

Comment:
The list might be helpful but you could use a more compact form like `< > " / \ | ? * :`